components/sections/OurWork.jsx

```diff
@@ -7,7 +7,6 @@ const OurWork = () => {
     <div className="flex flex-col mt-6 lg:mt-24 mb-7.5 lg:mb-8 ">
       <div>
       <Title content="Our Work" styles={'lg:ml-24  ml-0'}/>
```

Comment:
The unnecessary space between class names should be removed for consistency purposes